docs/fidesops/docs/development/ui_development.md

```diff
@@ -0,0 +1,28 @@
+# Admin UI
+
+Instructions on running the Admin UI and generating users for local development.
+
+## Running Locally
+
+To get started locally, clone the [FidesOps repository](https://github.com/ethyca/fidesops/), and ensure you have [Node.js](https://nodejs.org/en/download/) installed to run the application.
+### Creating the root user
+
+In the top-level `fidesops` directory, run `make user`.
+
+A series of prompts will walk you through creating a username and password. Passwords require 8 or more characters, upper and lowercase characters, a number, and a symbol. 
+
+This will create an Admin UI Root User that can be used to access additional [user endpoints](#managing-users).
+
+### Accessing the Control Panel
+
+From the root `fidesops` directory, run the following:
+``` sh
+    cd clients/admin-ui
+    npm run dev
+```
```

Comment:
They'll need to install all of the dependencies first. I do this all the time 🤦‍♂️ I try to run the project first and realize I didn't install anything yet

```suggestion
``` sh
    cd clients/admin-ui
    npm install
    npm run dev
```
```